applications/RANSApplication/custom_processes/rans_check_scalar_bounds_process.cpp

```diff
@@ -0,0 +1,110 @@
+//    |  /           |
```

Comment:
I think that this can be more efficiently implemented as a method in your already existent RANS utilities. I think that introducing a new class (2 more files in the repo) to check the maximum and minimum values of a model part variable is way too much.

Reply:
moved to python level, and removed this from c++ :)